tests/installation/test_installer.py

```diff
@@ -27,7 +27,7 @@
 from poetry.installation import Installer as BaseInstaller
 from poetry.installation.executor import Executor as BaseExecutor
 from poetry.installation.noop_installer import NoopInstaller
-from poetry.packages import Locker as BaseLocker
+from poetry.packages.locker import Locker as BaseLocker
```

Comment:
I feel like this import is a bit confusing since there is now a `BaseLocker` in that file. Can we clean it up at all?